docs-rtd/howto/index.md

```diff
@@ -25,6 +25,7 @@ manage-offers
 manage-units
 manage-secrets
 manage-machines
+manage-migrations
 ```
```

Comment:
I would fold this into Manage models > Migrate. A migration is not really an entity with a lifecycle of its own, like relations.